site-manage/user-picker/config/index.js

```diff
@@ -1,7 +1,7 @@
 // see http://vuejs-templates.github.io/webpack for documentation.
 var path = require('path')
 
-const sakaiServer = 'http://dev:8080'
+const sakaiServer = 'http://localhost:8080'
```

Comment:
What's going on here?

Reply:
This is a way of running a proxy infront of Sakai for development. I'd been using an additional hostname because of how I used docker, but I'm not doing this any more and using localhost is more standard.